Marlin/Marlin_main.cpp

```diff
@@ -8434,6 +8435,21 @@ inline void gcode_M211() {
   SERIAL_ECHOLNPAIR(" " MSG_Z, soft_endstop_max[Z_AXIS]);
 }
 
+/**
+ * M217: Send text to serial output
+ *
+ * Usage: M217 some text
+ * Text must be prefixed by "//" and this will be prepended if it is not present
```

Comment:
I read in the reprap wiki that this kind of output from the machine to the host should be prefixed that way, but I guess it's not really mandatory for all cases (`//` seems to be expected for action commands though).